psi4/src/psi4/occ/postprocessing.cc

```diff
@@ -113,22 +109,21 @@ void OCCWave::mp2_postprocessing(bool include_singles) {
     // Wavefunctions saved to variables are set on the wavefunction Py-side.
     // Other variables will be set Py-side via QCDB formulas, per a future Lori project.
     variables_["MP2 CORRELATION ENERGY"] = Emp2 - Escf;
-    variables_["SCS-MP2 CORRELATION ENERGY"] = 6.0/5.0 * Emp2AB + 1.0/3.0 * (Emp2AA + Emp2BB) + Emp2_t1;
+    variables_["SCS-MP2 CORRELATION ENERGY"] = 6.0 / 5.0 * Emp2AB + 1.0 / 3.0 * (Emp2AA + Emp2BB) + Emp2_t1;
```

Comment:
Isn't the 6/5, 1/3 already covered by the likes of https://github.com/psi4/psi4/pull/1789/files#diff-fd2a6d2dc2bc31f90b49a7bdcff983d6R181 ?

Reply:
They are. To make things even more "fun", the two Escsmp2 formulas aren't even the same. They differ on whether you include the T1 contribution in the SCS-MP2 formula.

I can either replace the wavefunction variables (advantage: less cluttered wavefunction) or replace this call with said variables (advantage: we don't need to construct the spin-component energy multiple times). I'll think over which approach seems better.

Reply:
fwiw, I went with including singles in qcdb https://github.com/qcdb/qcdb/blob/master/qcdb/qcvars/identities.py#L87

Reply:
After checking the original Grimme paper, I _suspect_ that SCS-MP2 is defined to exclude singles contribution, but it isn't explicit. Because QCDB, I'll include singles energy provisionally. Grimme is stopping by UGA in a couple weeks, so I'll see if I can ask him directly.

Reply:
iirc, Sherrill's SCS-CCSD paper _did_ include singles, hence qcdb's bias